website/src/data/users.tsx

```diff
@@ -2433,6 +2432,607 @@ const Users: User[] = [
     source: 'https://github.com/7Wate/wiki',
     tags: ['opensource', 'personal'],
   },
+  {
+    title: 'Discord API Types',
+    description: 'Discord API Types',
+    preview: null,
+    website: 'https://discord-api-types.dev/',
+    source: 'https://github.com/discordjs/discord-api-types/tree/main/website',
+    tags: ['opensource', 'versioning', 'large'],
+  },
+  {
+    title: 'Hanabi',
+    description: 'A list of Hanabi card game strategies',
+    preview: null,
+    website: 'https://hanabi.github.io/',
+    source: 'https://github.com/hanabi/hanabi.github.io',
+    tags: ['opensource'],
+  },
+  {
+    title: 'IsaacScript',
+    description: 'Write Binding of Isaac: Repentance mods with TypeScript',
+    preview: null,
+    website: 'https://isaacscript.github.io/',
+    source: 'https://github.com/IsaacScript/isaacscript/tree/main/packages/docs',
+    tags: ['opensource'],
+  },
+  {
+    title: 'Jest Preview',
+    description: 'Debug your Jest tests. Effortlessly.',
+    preview: null,
+    website: 'https://www.jest-preview.com/',
+    source: 'https://github.com/nvh95/jest-preview/tree/main/website',
+    tags: ['opensource'],
+  },
+  {
+    title: 'Paweł Kosiec',
+    description: 'Personal website and blog of Paweł Kosiec, Full-stack Cloud Developer.',
+    preview: null,
+    website: 'https://kosiec.dev/',
+    source: 'https://github.com/pkosiec/website',
+    tags: ['opensource','personal'],
+  },
+  {
+    title: 'Homarr',
+    description: 'Homarr is a simple and lightweight homepage for your server.',
+    preview: null,
+    website: 'https://homarr.vercel.app/',
+    source: 'https://github.com/ajnart/homarr/tree/docs',
+    tags: ['opensource'],
+  },
+  {
+    title: 'Wings',
+    description: 'Wings is a modern website-as-service for progressive campaigns',
+    preview: null,
+    website: 'https://wings.dev/',
+    source: null,
+    tags: ['product','i18n'],
+  },
+  {
+    title: 'Kishan Gajera',
+    description: 'Personal portfolio and blog of Kishan Gajera',
+    preview: null,
+    website: 'https://www.kgajera.com/',
+    source: 'https://github.com/kgajera/blog',
+    tags: ['opensource','personal'],
+  },
+  {
+    title: 'Harvest CLI',
+    description: 'A CLI for Harvest\'s time tracking software',
+    preview: null,
+    website: 'https://kgajera.github.io/hrvst-cli/',
+    source: 'https://github.com/kgajera/hrvst-cli/tree/main/website',
+    tags: ['opensource'],
+  },
+  {
+    title: 'Tremor',
+    description: 'Tremor Event Processing Engine',
+    preview: null,
+    website: 'https://www.tremor.rs/',
+    source: 'https://github.com/tremor-rs/tremor-www/',
+    tags: ['opensource','versioning','large'],
+  },
```

Comment:
It seems like our page snuck in twice. That's on me, sorry about that o.O
```suggestion
```

Reply:
Thanks, it was already added in https://github.com/facebook/docusaurus/pull/6015